antarest/storage/service.py

```diff
@@ -347,7 +345,7 @@ def _get_study(self, uuid: str) -> Study:
 
     def _assert_permission(
```

Comment:
This should now take an additional parameter (action: read,run,edit,delete,manage_permissions) to enforce the permission matrix.

Reply:
yes storage service permission is not yet implemented. I will do in other PR.